tools/figma_import/src/rendered.rs

```diff
@@ -169,8 +169,8 @@ fn render_vector(
                 if let Some(_imr) = &p.imageRef { /* */ }
             }
             rc.end_element()?;
-            return Ok(false);
         }
+        return Ok(false);
```

Comment:
I guess that is probably correct.
Too bad we don't have any test to make sure that this was not really intended (so that only one is run)